res/database/Default/user/sjaele.xml

```diff
@@ -314,7 +314,7 @@
 			</title>
 			<description>
 				<de>Der Darische Kalender unterteilt das Marsjahr in 24 Monate und enthält sämtliche Schalttage. Nur noch einen Tag Abweichung auf 12 000 Marsjahre.</de>
-				<en>The Darian Calendar divides the Mars year into 24 months and contains all leap days. Only one more day to 12,000 Mars years.</en>
+				<en>The Darian Calendar divides the Martian year into 24 months and includes all leap days. Deviation now only one day per 12 000 Mars years.</en>
```

Comment:
Usually you used "," for making large numbers more readable, here you removed it.

Reply:
git blame translation tool

yeah I prefer comma too